src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -387,10 +389,14 @@ public void addNotification(int accountId, int chatId, int msgId) {
                 builder.setGroup(GRP_MSG + "." + accountId);
             }
 
-            String accountAddr = dcContext.getConfig("addr");
+            String accountTag = dcContext.getConfig(CONFIG_PRIVATE_TAG);
+            if (accountTag.isEmpty() && context.dcAccounts.getAll().length > 1) {
+                accountTag = dcContext.getName();
+            }
+
             if (privacy.isDisplayContact()) {
                 builder.setContentTitle(dcChat.getName());
-                builder.setSubText(accountAddr);
+                builder.setSubText(accountTag);
```

Comment:
otherwise, a 3rd, empty line may be added, see NotificationCompat.java source.

it's also better style to just not set in case nothing is to set.

```suggestion
                if (!TextUtils.isEmpty(accountTag)) {
                    builder.setSubText(accountTag);
                }
```